x-pack/test_serverless/functional/test_suites/security/config.mki_only.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ loadTestFile }: FtrProviderContext) {
+  describe('serverless security UI', function () {
+    this.tags(['security-mki-only']);
```

Comment:
is there a configuration we need to coordinate with the	[qaf-tests](https://github.com/elastic/qaf-tests/blob/main/.buildkite/pipeline_generators/cloud/serverless/ftr-tests.py) repo to include config files by using the tag?

Reply:
I saw now: https://github.com/elastic/qaf-tests/pull/141